### ISSUE TITLE
Feature Request: Add network statistics / Information to Compass.

### ISSUE THREAD

**Issue by Starystars67** [mongodb-js/compass#2364]
## Feature Request
Please can you add a new tab to compass called network. Something like this:
![image](https://user-images.githubusercontent.com/15389482/126761071-e4f5ce64-5ed6-493e-ba92-a63de561aafe.png)

## Detailed Description
On this new tab/page it should display the information about the network such as if you are running a replica set, the status of each of the replica set nodes. which one is primary and so on. I do understand that there is the new webpage that offered a certain level of granularity however I was hoping that this would allow for you to potentially administrate your network using the shell all from within Compass.

## Context

The idea of this is to allow for network/database administrators to easily see the status of the network and be able to see if there is an issue from a glance. At the same time they also already have a shell to hand ready for their commands.
Personally I found it to become a small pain trying to get the link for the webpage each time I wanted to be able to pull up the data for any particular given network.  This would make this easier for me in my case and I am sure it will help others too.

## Possible Implementation

Put simply this would serve as an alternative and more advanced version of the new metrics webpage. providing the ability to run commands at the click of a button to open the shell.

I would be happy to attempt to implement it myself however I will require some level of help to understand the setup here if anybody is willing to.

**Comment by Anemy** [comment]
Hi @Starystars67 - apologies for the late reply on this.
We're tracking feature requests and future projects in our feedback forum: https://feedback.mongodb.com/forums/924283-compass - I haven't really seen a request like this in there, so it would be good to add it. The feedback forum allows for upvoting so we can gauge how many people are interested in something and it lets folks stay informed when projects and requests are worked on and completed.
It definitely makes sense to improve the transparency of what the cluster and configuration is that someone is connected to in Compass.
If you're still interested in contributing we've got a `Getting Started` section in our readme: 
https://github.com/mongodb-js/compass/blob/main/CONTRIBUTING.md#getting-started
